Automattic/jetpack
# VideoPress: Persist hover preview values on markup for video block

Fixes #29870

## Proposed changes:

* This PR exposes the relevant preview-on-hover data to the frontend by adding `data-preview-on-hover` to the block's rendered markup

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor and add a `videopress/video` block with a video
* Go to the `Poster and preview` panel
* Check the `Video preview on hover` toggle and change the hover start and loop duration values (important, as default values are not yet supported)
* Publish the post
* In the post page, find the video block in the browser inspector
* In its `figure` element, check that `data-preview-on-hover` has a value like `data-preview-on-hover='{"previewAtTime":4133,"previewLoopDuration":22627}'`

![2023-04-03_15-19-04](https://user-images.githubusercontent.com/8486249/229593822-b611b4d6-131b-4a2d-b709-67abb6abc433.png)
